Proj_i02/settings.py

```diff
@@ -102,16 +102,20 @@
 
 AUTH_PASSWORD_VALIDATORS = [
```

Comment:
This change is part of [#13](https://github.com/ncs-jss/Proj_i02/pull/13)
Merge it in ```inventory``` branch to update this PR.